pkg/api/controller_test.go

```diff
@@ -1150,6 +1197,133 @@ func TestController_DeleteBranchHandler(t *testing.T) {
 	})
 }
 
+func TestController_IngestRangeHandler(t *testing.T) {
+	const (
+		fromSourceURI           = "https://valid.uri"
+		uriPrefix               = "take/from/here"
+		fromSourceURIWithPrefix = fromSourceURI + "/" + uriPrefix
+		after                   = "some/key/to/start/after"
+		continuationToken       = "opaque"
+		prepend                 = "some/logical/prefix"
+	)
+
+	ctx := context.Background()
+	token := continuationToken
+
+	setup := func(count int, err error) (api.ClientWithResponsesInterface, *testutils.FakeWalker) {
```

Comment:
Consider using as a helper function - passing t *testing.T as argument and cal Helper, it will report the line in the test as failed and not this one for all the tests.

Reply:
Done.